.github/workflows/update-test-results.yml

```
@@ -597,29 +633,29 @@ jobs:
         steps:
             - uses: actions/checkout@v5
             - name: Download results
-              uses: actions/download-artifact@v5
-              with:
-                  pattern: 'result-*-*-*'
-                  merge-multiple: true
+                uses: actions/download-artifact@v5
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 636: mapping values are not allowed in this context

(syntax-check)